packages/trip-form/src/__mocks__/trimet.styled.js

```diff
@@ -50,6 +50,10 @@ const TriMetStyled = styled.div`
     padding: 4px 0px 0px;
     font-size: 10px;
     line-height: 12px;
+
+    &.active {
```

Comment:
We should not use classes like this with styled-components. Instead, the styled component should provide dynamic values based on the propsthat are sent. So this should be rewritten as:

```css
text-decoration: ${props => props.active && 'underline'}
```

Reply:
Fair point. Except that props refers to the top-level styled component, in which case is SettingsSelectorPanel, and props contains the props of that component only.